src/quo/components/drawers/action_drawers/style.cljs

```diff
@@ -12,11 +12,16 @@
    :flex-direction   :row})
 
 (defn container
-  [sub-label disabled?]
-  {:border-radius     12
-   :height            (if sub-label 56 48)
-   :opacity           (when disabled? 0.3)
-   :margin-horizontal 8})
+  [{:keys [sub-label disabled? state customization-color blur? theme]}]
+  (merge
+   {:border-radius     12
+    :height            (if sub-label 56 48)
+    :opacity           (when disabled? 0.3)
+    :margin-horizontal 8}
+   (when (= state :selected)
+     {:background-color (if blur?
+                          colors/white-opa-5
+                          (colors/resolve-color customization-color theme 5))})))
```

Comment:
Not a blocker: Since we use `merge` with only two map. We can refactor this to use `cond->` and it's a bit faster than merge too.

```suggestion
  (cond-> {:border-radius     12
           :height            (if sub-label 56 48)
           :opacity           (when disabled? 0.3)
           :margin-horizontal 8}

    (= state :selected)
    (assoc :background-color (if blur?
                          colors/white-opa-5
                          (colors/resolve-color customization-color theme 5)))))
```